tofu/geom/_GG.pyx

```diff
@@ -1099,7 +1099,7 @@ def _Ves_Vmesh_Tor_SubFromD_cython(double rstep, double zstep, double phistep,
     _st.vmesh_prepare_tab(lnp, sz_r, sz_z, sz_phi)
     indI = np.sort(indI, axis=1)
     indi_mv = indI
-    first_ind_mv = np.argmax(indI > -1, axis=1).astype(np.long)
+    first_ind_mv = np.argmax(indI > -1, axis=1).astype(int)
```

Comment:
Is this the new tuype introduced by more recent numpy / cython versions ?

Reply:
yes, it seems to be working on windows as well (Github Actions tests passing)